gcta/1.24.7/Dockerfile

```diff
@@ -0,0 +1,40 @@
+#################################################################
+# Dockerfile
+#
+# Version:          1
+# Software:         gcta
+# Software Version: 1.24.7
+# Description:      a tool for Genome-wide Complex Trait Analysis
+# Website:          http://cnsgenomics.com/software/gcta/index.html
+# Tags:             Genomics
+# Provides:         gcta 1.24.7
+# Base Image:       biodckr/biodocker
+# Build Cmd:        docker build --rm -t biodckrdev/gcta 1.24.7/.
+# Pull Cmd:         docker pull biodckrdev/gcta
+# Run Cmd:          docker run --rm -it biodckrdev/gcta <options> <files>
+#################################################################
+
+# Set the base image to Ubuntu
+FROM biodckr/biodocker
```

Comment:
same remarks, should use biocontainers base image and use LABEL for metadata (+ license info)